kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -38,7 +39,7 @@ handle_options() {
             usage
             exit 0
             ;;
-        --cloud) cloud_mode=1;;
+        --cloud) cloud_mode=1 ;;
```

Comment:
@Twey Weirdly, I ran the formatter again, and now it fixed the spacing here 🤣

Reply:
Scary!